src/components/row.js

```diff
@@ -25,7 +26,9 @@ function Row(props) {
   }, [props.districts]);
 
   const handleReveal = () => {
-    props.handleReveal(props.state.state);
+    if (typeof props.handleReveal === 'function') {
+      props.handleReveal(props.state.state);
+    }
   };
```

Comment:
You also did the same thing which I commented on this PR
https://github.com/covid19india/covid19india-react/pull/389/files/2f47962cff23be3c6a41bfbeca59861bc1a56f12#r400408957